app/pages/dashboard/PatientDrawerContent/useAGPImages/getQueries.js

```diff
@@ -0,0 +1,108 @@
+import _ from 'lodash';
+import { utils as vizUtils } from '@tidepool/viz';
+import utils from '../../../../core/utils';
+
+const chartPrefs = {
+  agpBGM: {
+    bgSource: 'smbg',
+  },
+  agpCGM: {
+    bgSource: 'cbg',
+  },
+  settings: {
+    touched: false,
+  },
+  excludedDevices: [],
+};
+
+const getQueries = (
```

Comment:
To create this parent function `getQueries`, I started at [this argument in a call to generatePDF()](https://github.com/tidepool-org/blip/blob/WEB-3336-tide-drawer/app/pages/patientdata/patientdata.js#L1061) and worked backwards up the callback chain to figure out how the `queries` arg is determined. 

Working backwards in the callback chain to get to the `queries` arg, I get to these 2 values which are defined by `this.state.bgPrefs` and `this.state.timePrefs`. Because they are held in component state, the process of how they get their value is a little nebulous.

From code reading and testing, it seems like `bgPrefs` represent the `clinic's` preferences for bg readings, while timePrefs represent the _patient's_ device preferences for time. Is this understanding correct?

I _believe_ they are defined [here in componentWillReceiveProps](https://github.com/tidepool-org/blip/blob/WEB-3336-tide-drawer/app/pages/patientdata/patientdata.js#L1694-L1718). Unfortunately, I can't simply copy and paste that code because it is coupled to component lifecycle, so I have written this batch of code to get us `bgPrefs` and `timePrefs`, and I think it works and is accurate. 

Would be awesome if you could help me double check this work to generate the `queries` arg when we get to code review. I'm a little nervous about it.